tests/dialects/test_bigquery.py

```diff
@@ -1272,3 +1273,19 @@ def test_json_object(self):
 
         with self.assertRaises(ParseError):
             transpile("SELECT JSON_OBJECT('a', 1, 'b') AS json_data", read="bigquery")
+
+    def test_parse_drop_schema(self):
```

Comment:
let's move all of these tests into test_parsers. they are very repetitive.

you can do a for look

for dialect in snowflake, bigquery, ...
 

additionally, don't really need to breakdown and test the ast like, just check the ast equality

ast == exp.Drop(...)